### PR TITLE
COL-262 Do not include current user in online whiteboard users count

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-262

### CODE REVIEW

**Comment by paulkerschen** [comment]
Assigning to @nicolaasmatthijs for review.

File: node_modules/col-whiteboards/tests/util.js [hunk 1]
```diff
@@ -68,6 +68,9 @@ var assertWhiteboard = module.exports.assertWhiteboard = function(whiteboard, op
     }
   } else {
     assert.ok(_.isFinite(whiteboard.online_count));
+    if (opts.expectedOnlineCount) {
```
Comment (nicolaasmatthijs) [inline node_modules/col-whiteboards/tests/util.js]: `expectedOnlineCount` should be documented in the JSDocs

File: node_modules/col-whiteboards/lib/api.js [hunk 2]
```diff
@@ -279,12 +279,12 @@ var getWhiteboards = module.exports.getWhiteboards = function(ctx, limit, offset
...
-    // Add the number of online users for each whiteboard. Note that a single user
-    // can have multiple sessions in the same whiteboard. This only counts as a single
-    // online user
+    // Add the number of online users for each whiteboard, excluding the current user.
+    // Note that a single user can have multiple sessions in the same whiteboard. This only
+    // counts as a single online user
     result.rows = _.map(result.rows, function(row) {
       row = row.toJSON();
-      row.online_count = _.chain(row.whiteboard_sessions).pluck('user_id').uniq().value().length;
+      row.online_count = _.chain(row.whiteboard_sessions).pluck('user_id').pull(ctx.user.id).uniq().value().length;
```
Comment (nicolaasmatthijs) [inline node_modules/col-whiteboards/lib/api.js]: There is probably enough on this line to format it to multiple lines:

```
_.chain(row.whiteboard_sessions).pluck('user_id')
                                                      .pull(ctx.user.id)
                                                      .uniq()
                                                      .value()
                                                      .length;
```

Comment (nicolaasmatthijs) [inline node_modules/col-whiteboards/lib/api.js]: The formatting of the above is a bit messed up, but you get the gist :)

**Comment by nicolaasmatthijs** [comment]
Looks good. Left 2 minor comments. Back to @paulkerschen for follow-up

**Comment by paulkerschen** [comment]
Amended to address comments, back to @nicolaasmatthijs

**Comment by nicolaasmatthijs** [comment]
This comment might have gotten lost in https://github.com/ets-berkeley-edu/collabosphere/pull/126:

```
@paulkerschen : Would you mind keeping code review follow-up commits separate from the original commit (i.e., not squash them)? This makes it easier to just review the code review follow-up changes.
```

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged.

**Comment by paulkerschen** [comment]
Missed the comment about follow-up commits the first time, @nicolaasmatthijs - no problem.

**Pull request merged**